homeassistant/components/openuv/sensor.py

```diff
@@ -42,7 +42,7 @@
 UV_LEVEL_MODERATE = "Moderate"
 UV_LEVEL_LOW = "Low"
 
-SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
+SENSOR_DESCRIPTIONS = (
```

Comment:
```suggestion
SENSOR_DESCRIPTIONS: tuple[SensorEntityDescription, ...] = (
```

Reply:
Why? `mypy` doesn't complain without this, which means the type checker can infer it.

Reply:
It's a nice to have, but yes not strictly necessary.